### PR TITLE
Fixed CX20756 files for InsanelyDeepak version

### PR DESCRIPTION
**Pull request by STiXzoOR**
Also in order for the External Mic to work, it needs Codec Commander with a custom SSDT file. Attached below if needed.

[SSDT-CX20756.dsl.zip](https://github.com/acidanthera/AppleALC/files/2475793/SSDT-CX20756.dsl.zip)

### CODE REVIEW

**Comment by STiXzoOR** [comment]
1. Didn’t see that coming. Well i edited the info.plist through xcode. What is the right way to edit the plist without breaking it?
2. Actually we were working together to make it work in the past and in the end we sorta got it working through AppleHDA patching. I did some tests on my own and fixed the issues and everything is working now. That’s why i wanted to update the files.

**Comment by vandroiy2013** [comment]
1. Just build kext before making pull request. It will automatically fix XML structure. 
2. I understand. But nevertheless try to contact @insanelydeepak. I will not make changes without his permission.

File: Resources/PinConfigs.kext/Contents/Info.plist [hunk 1]
```diff
@@ -6079,7 +6079,7 @@
 					<key>Codec</key>
 					<string>InsanelyDeepak - Conexant CX20756</string>
 					<key>CodecID</key>
-					<integer>351359247</integer>
+					<integer>351359252</integer>
```
Comment (vit9696) [inline Resources/PinConfigs.kext/Contents/Info.plist]: Is this change intentional? I get a feeling it may not be.

**Comment by vit9696** [comment]
@insanelydeepak besides the pinconfigs, do the changes look fine?

File: Resources/PinConfigs.kext/Contents/Info.plist (hunk 1 above)
Comment (STiXzoOR) [reply Resources/PinConfigs.kext/Contents/Info.plist]: well the CodecID in PinConfigs is the wrong one so i corrected it with the right one

**Comment by insanelydeepak** [comment]
Hi 
sorry guys i was away and github stopped stoped notifying me 
yeah there was mistake in codecID in my repo too , did fix long ago but not pushed it yet 

you can use below files to make changes 
https://github.com/insanelydeepak/cloverHDA-for-Mac-OS-Sierra-10.13/blob/master/Laptop/Conexant%20CX20756.zip


thank you
